simulation_parameters/SimulationParameters.cs

```diff
@@ -94,6 +95,9 @@ public override void _Ready()
         gameCredits =
             LoadDirectObject<GameCredits>("res://simulation_parameters/common/credits.json");
 
+        PatchMapNameGenerator = LoadDirectObject<PatchMapNameGenerator>(
+            "res://simulation_parameters/microbe_stage/patch_syllabels.json");
```

Comment:
"syllable"

Reply:
The typo is still here? Why was this marked as resolved?

Reply:
huh